thewalrus/internal_modes/fock_density_matrices.py

```diff
@@ -139,16 +139,44 @@ def _density_matrix_single_mode(
 
     rho = rho[:cutoff, :cutoff]
 
-    if normalize:
-        return rho / np.trace(rho).real
     return rho
 
 
+def check_probabilities(probs, atol=1e-08):
+    """
+    Convinience function for checking that the input is close enough to a probability distribution.
+
+    Args:
+        probs (array): probabilities to be tested.
+        atol (array): absolute tolerance relative to the normalization.
```

Comment:
```suggestion
        atol (float): absolute tolerance relative to the normalization.
```
Or is this intentionally an array so you can have different tolerances for each probability?

Reply:
typo